### DISCUSSION TITLE
Unable to download 100k images

### DISCUSSION THREAD

**Discussion by alabsihuda** [bdd100k/bdd100k#254] [Q&A]
Hi. I am getting the following error while trying to download the 100k images.
![100k image](https://user-images.githubusercontent.com/89980562/173724646-e6f36c01-d94a-4737-8877-185a28d3d2ce.PNG)

**Comment by hankplease** [comment]
It happens sometime. I failed to download the dataset in the morning,but it works some hours later.

**Reply by alabsihuda** [reply]
I have tried several times at different times but the same error

**Comment by amita-kapoor** [comment]
I am facing a similar issue, I am unable to download- the download starts, but after about 30 minutes, it fails. I have tried different browsers (FireFox, Chrome, Brave) and OS (OX X, Ubuntu 18.04)

**Comment by PDD0911-HCMUS** [comment]
I still get this issues. Do anybody solve this issues
